pkg/image/filefmt.go

```diff
@@ -46,6 +46,30 @@ var knownHeaders = Headers{
 		SizeOff: 0,
 		SizeLen: 0,
 	},
+	"vmdk": Header{
+		Format:      "vmdk",
+		magicNumber: []byte{0x4B, 0x44, 0x4D, 0x56},
```

Comment:
Why not have consts for the magic numbers? each of them appears 3 times.

Reply:
here are some ideas:
https://qvault.io/2019/10/21/golang-constant-maps-slices/

Reply:
@maya-r decided not to fix this one?

Reply:
I did something a bit different and used meaningful strings

Reply:
nice! but having every meaningful string x 3 is a must?